src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -223,11 +224,19 @@
     <HelixPreCommand Include="export __CrashDumpFolder=$HELIX_DUMP_FOLDER" Condition=" '$(TargetOS)' != 'OSX' " />
     <HelixPreCommand Include="export __CrashDumpFolder=/cores" Condition=" '$(TargetOS)' == 'OSX' " /> <!-- Helix doesn't specify the dump folder for OSX 10.14, so we need to manually specify it. Tracked by dotnet/core-eng#7872 -->
     <HelixPreCommand Include="cat $__TestEnv" />
-
     <HelixPreCommand Include="sudo bash -c 'echo $HELIX_DUMP_FOLDER/core.%u.%p > /proc/sys/kernel/core_pattern'" Condition=" '$(TargetOS)' != 'OSX' " />
     <HelixPreCommand Include="ulimit -c unlimited" Condition=" '$(TargetOS)' == 'OSX' " />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetsWindows)' != 'true' and $(RuntimeMode) == 'interpreter' " >
```

Comment:
You could just put these to in the `ItemGroup` above by conditioning the individual Item inclusion as we do in other commands.

```xml
<HelixPreCommand Include="...." Condition="'$(RuntimeMode)' == 'interpreter'" />
```

Also, `$(RuntimeMode)` needs to be enclosed in `'` in the condition.